fluentd/vendored_gem_src/fluentd/lib/fluent/plugin/in_tail.rb

```diff
@@ -854,9 +892,18 @@ def on_notify(stat)
           begin
             if @inode != inode || fsize < @fsize
               @on_rotate.call(stat)
+              inodelastfsize_map[@inode]=@fsize
+              @totalbytesavailable=@fsize
```

Comment:
What is the purpose of maxfsize if you are using the fsize for logloss computation?

Reply:
maxfsize is taken out now as it is not required to be monitored.